client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -8,11 +8,14 @@
 </license>
 
 <template>
-  <div>
+  <div
+    :class="{ 'top-0 left-0 w-full h-full fixed z-[2200] bg-white': isFullscreen }">
     <dp-sticky-element
       border
-      class="u-pv-0_5">
-      <div class="flex items-start space-inline-s">
+      class="py-4"
+      :class="{ 'fixed top-0 left-0 w-full h-[15%] px-4': isFullscreen }">
+      <div
+        class="flex items-center justify-between mb-2">
```

Comment:
I'd suggest to have the filters left aligned, along with the custom search. this way it gets clear that "zurücksetzen" refers to both search and filter.

to right align an element (the fullscreen button) within a flex container even if other stuff is aligned left, you can add `ml-auto` to it. that way you do not need an additional container. Pseudo-Code:

```
<div class="flex items-start space-x-1 mb-2">
    <custom-search />
   <div class="flex bg-color--grey-light-2 rounded-md space-inline-xs">
      Filters
    </div>
    <resetFilter-button />
    <editorFullscreen-button class="ml-auto" />
</div>
```

Reply:
ok, done here https://github.com/demos-europe/demosplan-core/pull/2979/commits/a842dfe8552b4c374486dcc41c97d3f971f4e2b2